src/components/TheSearchBar.vue

```diff
@@ -3,7 +3,7 @@
     <input
       v-model="query"
       class="search-bar__input"
-      placeholder="Search accounts, transactions, names, contracts, oracles, state channels, keyblocks, and microblocks"
+      placeholder="Search anything on aeScan"
```

Comment:
please check the zooming issue on iPhone @Liubov-crypto

Reply:
> please check the zooming issue on iPhone @Liubov-crypto

Still reproducible for me on iPhone 13 in Safari browser ver. 17.1.2

Reply:
I tried to apply possible fix. Please try again now